dll/win32/shell32/CDefView.cpp

```diff
@@ -1950,7 +1954,7 @@ LRESULT CDefView::OnCommand(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHand
             {
                 HMENU Dummy = NULL;
                 MenuCleanup _(m_pFileMenu, Dummy);
-                InvokeContextMenuCommand(m_pFileMenu, dwCmdID);
+                InvokeContextMenuCommand(m_pFileMenu, dwCmdID, NULL);
```

Comment:
Does this show a menu or remove it? (and does a similar "FIXME" to the one above apply there too?)

Reply:
As the comment indicates, this is the 'File' menu.